### ISSUE TITLE
Benchmarks

### ISSUE THREAD

**Issue by pickfire** [rust-dc/fish-manpage-completions#84]
Benchmark performance against python `fish-manpage-completions`, can use https://github.com/sharkdp/hyperfine for this, we could even steal from https://github.com/XAMPPRocky/tokei for https://github.com/XAMPPRocky/tokei/blob/master/benchmark.sh

**Comment by necaris** [comment]
Now that the first version of things is "done"... this seems to have gained relevance?

**Comment by pickfire** [comment]
Yes, I have some minor benchmarks here IIRC and we should also try to update our code to match the latest python code but this is good enough to be done now, for a one-to-one core comparison.
